docs/pages/access-controls/guides/dual-authz.mdx

```diff
@@ -13,23 +13,59 @@ Here are the most common scenarios:
 Let's set up Teleport's access requests to require approval of
 two team members for a privileged role `dbadmin`.
 
-<Admonition
+<Notice
   type="warning"
-  title="Version Warning"
 >
   This guide requires a commercial edition of Teleport. The open source
-  edition of Teleport only supports [Github](../../setup/admin/github-sso.mdx) as
+  edition of Teleport only supports [GitHub](../../setup/admin/github-sso.mdx) as
   an SSO provider.
-</Admonition>
+</Notice>
 
 <Admonition title="Note" type="tip">
   The steps below describe how to use Teleport with Mattermost. You can also [integrate with many other providers](../../enterprise/workflow/index.mdx).
 </Admonition>
 
 ## Prerequisites
 
-- Installed [Teleport Enterprise](../../enterprise/introduction.mdx) or [Teleport Cloud](../../cloud/introduction.mdx) >= 7.0.0
-- [Tctl enterprise admin tool](https://goteleport.com/teleport/download) >= 7.0.0
+<Tabs>
```

Comment:
Why not use the new partial here?

Reply:
The new partial includes a `TabItem` for open source Teleport. I thought about creating a similar partial for commercial-only guides, but wanted to wait until I identified more uses for that partial before creating it.